api/src/test/resources/org/openmrs/api/db/hibernate/include/HibernateAlertDAOTestDataSet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+-->
+<dataset>
+	<alert alert_Id="12345" text="Testing the hibernate DAO"/>
```

Comment:
For this,i am trying to get data from this newly created data set were i am specifying the alert with an id and its text,but i keep on getting .NoSuchTableException: alert 
 as seen here,https://pastebin.com/9J6zfFsn   Any thoughts on this cc  @ibacher  @dkayiwa

Reply:
@HerbertYiga That's because the name of the table for the `Alert` class is `notification_alert`. See [the relevant Hibernate mapping file](https://github.com/openmrs/openmrs-core/blob/4f10708faa23cf6ff90fd6fdff4de9c84df732eb/api/src/main/resources/org/openmrs/notification/db/hibernate/Alert.hbm.xml#L18). Also not that, at a minimum, you probably need to set `creator`, `satisfied_by_any`, `and `alert_read` and should set `uuid`.